react-backend/client/src/cb-admin/pages/ConfirmPassword.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { CbAdmin } from '../../api';
+import { Form, FormSection, PrimaryButton } from '../../shared/components/form/base';
+import { Heading, Paragraph, Link } from '../../shared/components/text/base';
+import LabelledInput from '../../shared/components/form/LabelledInput';
+import { FlexContainerCol } from '../../shared/components/layout/base';
+
+
+const SubmitButton = PrimaryButton.extend`
+  height: 4em;
+  width: 90%;
+`;
+
+const CenteredParagraph = Paragraph.extend`
```

Comment:
You use the `.extend` here but not in ResetPassword or ForgotPassword, is that intentional?

Reply:
No, just oversight, will fix it in the subsequent PR